cmd/devices_list.go

```diff
@@ -0,0 +1,92 @@
+// Copyright 2021 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/mendersoftware/mender-cli/client/devices"
+)
+
+// const (
+// 	argDetailLevel = "detail"
+// )
+
+var devicesListCmd = &cobra.Command{
+	Use:   "list",
+	Short: "List Mender Devices from the Mender server.",
```

Comment:
maybe rephrase a bit? suggested: "Get a list of devices from the Mender server"

Reply:
Fixed also similar sentence in artifacts_list